Seafile.json

```
@@ -0,0 +1,34 @@
+{
+    "Seafile-Official": {
```

You can simply the rock-on's name to simly _Seafile_ as it is the only rock-on for this project. Furthermore, the fact that it uses the official Seafile image would already be detailed if you accept the suggestion below for including this information in the "_Description_" field.

```suggestion
    "Seafile": {
```